lib/id3c/cli/redcap.py

```diff
@@ -362,6 +362,13 @@ def report(self, report_id: str, raw: bool = False) -> List[Dict[str, Any]]:
         return self._fetch('report', parameters)
 
 
+    def metadata(self) -> List[Dict[str, str]]:
+        """
+        Fetch the REDCap project metadata.
+        """
+        return self._fetch('metadata', {})
+
+
```

Comment:
There is an existing `fields` property which provides this, and I think "fields" is a better/more precise name than "metadata".

Reply:
Oops! I overlooked that entirely. Agree on our name being more precise. I'll drop this method.